express/routes/users.ts

```diff
@@ -0,0 +1,81 @@
+import { Request, Response, Router } from "express";
+import express from "express";
+import * as fs from "fs";
+import validateUserExists from "../middleware/IfExist.mdware";
+import createValidation from "../middleware/createValidation.mdware";
+import { IUser } from "../types/IUser";
+const router: Router = express.Router();
+
+router.get("/", async (req: Request, res: Response<IUser[]>) => {
+  const users: Array<IUser> = await JSON.parse(
+    fs.readFileSync("./data/users.json", "utf8")
```

Comment:
await should be used on async methods...

Reply:
fix